common/eq_packet_structs.h

```diff
@@ -1671,6 +1671,68 @@ struct GuildUpdate_Struct {
 	GuildsListEntry_Struct entry;
 };
 
+struct GuildMemberAdd_Struct {
+	/*000*/ uint32 guild_id;
+	/*004*/ uint32 unknown04;
+	/*008*/ uint32 unknown08;
+	/*012*/ uint32 unknown12;
+	/*016*/ uint32 level;
+	/*020*/ uint32 _class;
+	/*024*/ uint32 rank;
+	/*028*/ uint32 spirt; //not confirmed single byte 0x000000FF
```

Comment:
This field is a padded bool that shadows the `Display Guild Name` permission of the guild for the added player's rank (Recruit?) to show the guild name sprite above the player's head.

It might be unused now in favor of the AT_GuildShow stat change msg that updates the name sprite explicitly but would need to double check and to be safe if it's not complicated to add we should fill it with the guild's current permission value for recruits.

Reply:
hg, I looked at this closer and GuildShow is an RoF item and that type is not available in UF.  Any suggestions on backwards compatibility please let me know.  Thanks!

Reply:
There's probably not a way to implement the permission for older clients without getting hacky and sending a fake guild id -1 update for their player to everyone (which might break something else, would have to look closer). I'd ignore it for now on older clients and just send it for rof2.

This specific field doesn't appear to be used anyway from what I see and might just be for server side, but it should be easy to update it to match what live does. The actual display update for the permission change is done through the GuildShow update. The hacky solution for older clients can be investigated in another patch later.

Reply:
Thanks.  I will update with this approach. Thanks again for the insight.

Reply:
Looking at it a little more for GuildShow specifically, older clients could bail out in the translators when handling `OP_SpawnAppearance` if the type isn't supported (don't forget to delete the packet if doing this).  We might need to do this eventually for other (non-guild related) types.

Reply:
@hgtw, refactored the sendappearance functionality for GuildShow and updated the translators for UF/TI to discard the that type.  Appreciate your thoughts if you have a moment.

Reply:
Updated the db index as well

Reply:
and below spirit item